activerecord/test/cases/signed_id_test.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require "cases/helper"
+require "models/account"
+require "models/company"
+
+SIGNED_ID_VERIFIER_TEST_SECRET = "This is normally set by the railtie initializer when used with Rails!"
+
+ActiveRecord::Base.signed_id_verifier_secret = SIGNED_ID_VERIFIER_TEST_SECRET
+
+class SignedIdTest < ActiveRecord::TestCase
+  fixtures :accounts
+
+  setup { @account = Account.first }
+
+  test "find signed record" do
+    assert_equal @account, Account.find_signed(@account.signed_id)
+  end
+
+  test "find signed record with a bang" do
+    assert_equal @account, Account.find_signed(@account.signed_id)
```

Comment:
```suggestion
    assert_equal @account, Account.find_signed!(@account.signed_id)
```

Needs a little more bang to comport with the test name 😄

Reply:
lol